apps/files/src/views/FileReferencePickerElement.vue

```diff
@@ -52,7 +52,7 @@ export default {
 				{
 					label: t('files', 'Choose'),
 					type: 'primary',
-					callback: (node) => this.submit(node.fileid),
+					callback: (nodes) => { console.warn('got', nodes); this.submit(nodes[0].fileid) },
```

Comment:
```suggestion
					callback: (nodes) => { this.submit(nodes[0].fileid) },
```

or log at debug level but with a more clear message?

Reply:
Agreed, just change the log with a clearer message and then it LGTM :)

Reply:
Done